drivers/vdpa/virtio/virtio.c

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022 NVIDIA Corporation & Affiliates
+ */
+
+#include <unistd.h>
```

Comment:
@kailiangz1 
Can you please take the whole code of this patch out of drivers/vdpa directory?

it should be in drivers/virtio or if its library it should be in lib/virtio.